worksheet_B/Mandelbrot.cpp

```diff
@@ -18,38 +138,67 @@ int main()
 	for (int pixelY = 0; pixelY < image.height(); pixelY++)
 	{
 		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
+		double y0 = (maxY - minY) * (pixelY / 800.0) + minY;
 
-		for (int pixelX = 0; pixelX < image.width(); pixelX++)
+		for (int pixelX = 0.0; pixelX < image.width(); pixelX++)
```

Comment:
`pixelX` is an `int` -- doesn't make sense to assign `0.0` to it (the compiler will automatically convert it back to integer `0`)